src/asfvideo.cpp

```diff
@@ -46,15 +46,11 @@ bool AsfVideo::GUIDTag::operator==(const AsfVideo::GUIDTag& other) const {
 }
 
 AsfVideo::GUIDTag::GUIDTag(const uint8_t* bytes) {
-  std::memcpy(&data1_, bytes, DWORD);
-  std::memcpy(&data2_, bytes + DWORD, WORD);
-  std::memcpy(&data3_, bytes + DWORD + WORD, WORD);
+  data1_ = (static_cast<uint32_t>(bytes[3]) << 24) | (static_cast<uint32_t>(bytes[2]) << 16) |
```

Comment:
Please could you use `getULong` (from `types.hpp`) here? I'd like to minimize the number of places where we do this kind of byte-swapping code.

Reply:
I changed it for data1_.